### DISCUSSION TITLE
Dynamic Thresholding?

### DISCUSSION THREAD

**Discussion by virt-10** [comfyanonymous/ComfyUI#615] [General]
Is [Dynamic Thresholding](https://github.com/mcmonkeyprojects/sd-dynamic-thresholding) possible? I like using this with a low mimic scale to get realistic looking images, but I have not found a way to do this in ComfyUI. Will this be added?

**Comment by comfyanonymous** [comment]
You can try the custom node here: https://github.com/comfyanonymous/ComfyUI_experiments

It's similar to dynamic thresholding.

**Reply by virt-10** [reply]
Thanks. Works nicely.
![ComfyUI_00014_](https://user-images.githubusercontent.com/121334908/236706835-8912ae7a-f3a2-41a3-bba6-9278808ca728.png)
